sdks/java/io/hadoop-input-format/hifio/src/main/java/org/apache/beam/sdk/io/hadoop/inputformat/HadoopInputFormatIO.java

```diff
@@ -845,7 +845,8 @@ public Double getFractionConsumed() {
        */
```

Comment:
Removed annotation VisibleForTesting.

Closed reader
defined set of known immutables in class initialization, which will be reused.